Makefile

```diff
@@ -28,11 +28,18 @@ clean:
 	@$(call banner,Cleaning)
 	rm -rf ./$(build_dir)
 
-.PHONY: lint
-lint:
+.PHONY: lint-go
+lint-go:
 	@$(call banner,Running golangci-lint)
 	@golangci-lint run
 
+.PHONY: lint-shell
```

Comment:
do  we use `lint-shell` and `lint-go` targets so that we can run locally what will be run in the pipeline?

Reply:
Yeah, pretty much. I don't love the fact that the github actions model kind of discourages you from using the same make targets that you're using locally in the pipeline. It feels like it's easier to use a pre-packaged Github action as part of your pipeline, but then you introduce the possibility for something that works in your local but is broken in Github actions, or visa versa